bucket/pass-winmenu-nogpg.json

```
@@ -0,0 +1,15 @@
+{
+    "version": "1.9",
+    "homepage": "https://github.com/Baggykiin/pass-winmenu",
+    "description": "A simple, easy-to-use password manager for Windows that follows the philosophy of (and is compatible with) the Linux password manager [pass](https://www.passwordstore.org/).",
```

Do not use markdown syntax in notes. It's not for nothing.

```suggestion
    "description": "A simple, easy-to-use password manager for Windows that follows the philosophy of (and is compatible with) the Linux password manager pass.",

```

---

There is no need for url at all. If you want it, just add it to normal brackets surrounded with `'` to keep clickable url in terminal.